Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift

```diff
@@ -147,6 +147,36 @@ class LazyTensorOperation: TensorOperation {
         }
     }
 
+    func outputName(at index: Int) -> String {
+        precondition(index < outputCount,
+            "Output index out of bounds when getting outputName.")
+        var ssaName = ""
+        if let id = self.id {
+            ssaName = "%\(id)"
+        } else {
+            ssaName = "%\(ObjectIdentifier(self))"
+        }
+        if outputCount > 1 {
+            ssaName += ".\(index)"
+        }
+        return ssaName
+    }
+
+    var outputName: String {
+        switch outputCount {
+        case 0: return ""
+        case 1: return "\(outputName(at: 0))"
+        default:
+            do {
+                let outputNames = (0..<outputCount).map {
```

Comment:
Use `.lazy` so it won't create an array first.

```suggestion
                let outputNames = (0..<outputCount).lazy.map {
```

Reply:
Nice.. Thanks! Done.